### PR TITLE
Changes to umbrella and changelog

### PR DESCRIPTION
**Pull request by pragatimodi**

### CODE REVIEW

File: FirebaseAuth/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# 10.9.0
```
Comment (paulb777) [inline FirebaseAuth/CHANGELOG.md:1]: Next version is 10.12.0. Firebase products share a single version.

File: FirebaseAuth/Sources/Public/FirebaseAuth/FirebaseAuth.h [hunk 2]
```diff
@@ -51,3 +51,4 @@
 #import "FIRPhoneMultiFactorInfo.h"
 #import "FIRTOTPMultiFactorAssertion.h"
 #import "FIRTOTPMultiFactorGenerator.h"
+#import "FIRTOTPSecret.h"
```
Comment (paulb777) [inline FirebaseAuth/Sources/Public/FirebaseAuth/FirebaseAuth.h:54]: It looks likes there are two additional new public headers in https://github.com/firebase/firebase-ios-sdk/tree/totp-release-0413/FirebaseAuth/Sources/Public/FirebaseAuth.  Were those two headers' APIs approved as well? If so, they should also be here.

Comment (pragatimodi) [reply FirebaseAuth/Sources/Public/FirebaseAuth/FirebaseAuth.h:54]: Yes

```
#import "FIRTOTPMultiFactorAssertion.h"
#import "FIRTOTPMultiFactorGenerator.h"
#import "FIRTOTPSecret.h"
```

are the three headers present in this file. I believe, `FIRTOTPSecret` was the only one added in this PR, the other two were already existing.

Comment (paulb777) [reply FirebaseAuth/Sources/Public/FirebaseAuth/FirebaseAuth.h:54]: I see. Thanks.

**Review by paulb777 — APPROVED** [review]

**Review by Xiaoshouzi-gh — APPROVED** [review]

**Pull request merged**